src/interfaces/IERC7540.sol

```diff
@@ -23,83 +23,100 @@ interface IERC165 {
 }
 
 interface IERC7540DepositReceiver {
-    /// @notice Handle the receipt of an deposit Request
-    /// @dev The ERC7540 smart contract calls this function on the recipient
-    ///  after a `requestDeposit`. This function MAY throw to revert and reject the
-    ///  request. Return of other than the magic value MUST result in the
-    ///  transaction being reverted.
-    ///  Inspired by https://eips.ethereum.org/EIPS/eip-721
-    ///  Note: the contract address is always the message sender.
-    /// @param _operator The address which called `requestDeposit` function
-    /// @param _sender The address which funded the `asssets` of the Request (or message sender)
-    /// @param _receiver The owner of this Request
-    /// @param _requestId The RID identifier of the Request which is being received
-    /// @param _data Additional data with no specified format
-    /// @return `bytes4(keccak256("onERC7540DepositReceived(address,address,address,uint256,bytes)"))`
-    ///  unless throwing
-    function onERC7540DepositReceived(address _operator, address _sender, address _receiver, uint256 _requestId, bytes _data) external returns(bytes4);
+    /**
+     * @dev Handle the receipt of an deposit Request
+     *
+     * The ERC7540 smart contract calls this function on the recipient
+     * after a `requestDeposit`. This function MAY throw to revert and reject the
+     * request. Return of other than the magic value MUST result in the
+     * transaction being reverted.
+     * Inspired by https://eips.ethereum.org/EIPS/eip-721
+     *
+     * Note: the contract address is always the message sender.
+     *
+     * @param _operator The address which called `requestDeposit` function
```

Comment:
can't these params be remapped to 
`_operator` -> `_sender`
_sender -> '_owner'

to match rest of spec

Reply:
I guess it's operator here to match the https://github.com/ethereum/ercs/blob/master/ERCS/erc-721.md spec. We are also not using `sender` in the rest of the interface anymore.